### ISSUE TITLE
How is node['logstash']['instance'][name] populated? 

### ISSUE THREAD

**Issue by luntzel** [lusis/chef-logstash#327]
Installation is bombing out on this for both server and agent: 

Chef::Application.fatal!("attribute hash node['logstash']['instance']['#{name}'] must exist.") if node['logstash']['instance'][name].nil?

chef-client 11.12.8
ubuntu 12.04

**Comment by algesten** [comment]
I'm completely lost with this. The above linked example is gone. The wiki pages are outdated. Can anyone post a working example?

**Comment by MerlinGuy** [comment]
So what are you trying to do?  If you just need to get past the "node['logstash']['instance'][name].nil" error and you are trying to install the logstash server all that is needed is to provide the json snippet logstash.instance.server somewhere.  This can be done in the attributes->default.rb file or in the chef-client input file like I describe above.

My problem was that I could not find an example of an LWRP that specifically installed the logstash server so I had to write my own recipe plus I wrote one that installed the logstash-forwarder.
